src/app/components/InventoryItem/index.css

```diff
@@ -0,0 +1,3 @@
+.crossed-off {
```

Comment:
Aren't you working with styled components?

Reply:
No, I ditched them :D